package/Dockerfile

```diff
@@ -8,80 +13,91 @@ ENV CRIDOCKERD_VERSION=0.3.15
 ENV RANCHER_CONFD_VERSION=v0.16.6
 ENV KUBECTL_VERSION=v1.28.11
 
-LABEL maintainer "Rancher Labs <support@rancher.com>"
 ARG ARCH=amd64
 ENV DOCKER_URL_amd64="https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz" \
     DOCKER_URL_arm64="https://download.docker.com/linux/static/stable/aarch64/docker-${DOCKER_VERSION}.tgz" \
     DOCKER_URL="DOCKER_URL_${ARCH}"
 ENV CRIDOCKERD_URL="https://github.com/Mirantis/cri-dockerd/releases/download/v${CRIDOCKERD_VERSION}/cri-dockerd-${CRIDOCKERD_VERSION}.${ARCH}.tgz"
-RUN apk -U upgrade \
-    && apk -U --no-cache add bash \
-    && rm -f /bin/sh \
-    && ln -s /bin/bash /bin/sh
-RUN apk -U --no-cache add curl wget ca-certificates tar sysstat acl\
-    && mkdir -p /opt/rke-tools/bin /etc/confd \
-    && curl -sLf "https://github.com/rancher/confd/releases/download/${RANCHER_CONFD_VERSION}/confd-${RANCHER_CONFD_VERSION}-linux-${ARCH}" > /usr/bin/confd \
-    && chmod +x /usr/bin/confd \
-    && curl -sLf "${!DOCKER_URL}" | tar xvzf - -C /opt/rke-tools/bin --strip-components=1 docker/docker \
-    && curl -sLf "${CRIDOCKERD_URL}" | tar xvzf - -C /opt/rke-tools/bin --strip-components=1 cri-dockerd/cri-dockerd \
-    && chmod +x /opt/rke-tools/bin/cri-dockerd \
-    && curl -sLf "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl" > /usr/local/bin/kubectl \
-    && chmod +x /usr/local/bin/kubectl \
-    && apk del curl
-
-RUN mkdir -p /opt/cni/bin
-RUN wget -q -O - https://github.com/containernetworking/plugins/releases/download/${CNI_PLUGINS_VERSION}/cni-plugins-linux-${ARCH}-${CNI_PLUGINS_VERSION}.tgz | tar xzf - -C /tmp \
-    && wget -q -O /tmp/flannel "https://github.com/flannel-io/cni-plugin/releases/download/${FLANNEL_VERSION}/flannel-${ARCH}" \
-    && chmod +x /tmp/flannel
+
+RUN mkdir -p \
+    /chroot \
+    /chroot/opt/rke-tools/bin \
+    /chroot/etc/confd \
+    /opt/cni/bin
+
+COPY --from=final-base / /chroot/
+
+RUN rm -f /chroot/bin/sh && ln -s /chroot/bin/bash /chroot/bin/sh
+
+RUN zypper refresh && \
+    zypper -n in wget file
```

Comment:
Wonder if we should start adding as a precaution the FIPS related packages - `openssl` and `patterns-base-fips` - as in https://github.com/rancher/rancher/pull/46575.

Reply:
Agree in skipping it for now. Let's wait for the Hostbusters team to review this.

Reply:
Had another look at this and still same issues building BCI with FIPS. My understanding is that the image we were using wasn't FIPS compliant. Is this needed for `rke-tools`?

Reply:
Agree ^, let's leave those packages out and add later when/if needed.

Reply:
Agree ^, let's leave those packages out and add later when/if needed.